pages/common/pio-settings.md

```diff
@@ -0,0 +1,20 @@
+# pio settings
+
+> View and modify PlatformIO settings.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/cmd_settings.html>.
+
+- List all PlatformIO settings:
+
+`pio settings get`
+
+- Show a specific PlatformIO setting:
```

Comment:
```suggestion
- Display the value of a PlatformIO setting:
```

Reply:
It doesn't only print the value, but a nice table:
```
Name          Current value [Default]    Description
------------  -------------------------  ------------------------------------
enable_cache  Yes                        Enable caching for HTTP API requests
```

Reply:
👍🏻 check again